### ISSUE TITLE
About deep deform dataset preprocessing.

### ISSUE THREAD

**Issue by phamtrongthang123** [USTC3DV/NDR-code#2]
Thank you for sharing your wonderful work. I was able to run the killing fusion sample with your instruction and am starting to move on to deep deform dataset. However, as you know Deep deform dataset does not provide mask for all frames (in train and val only a few main frames, and test does not provide any). So how did you create enough masks for the config file (./confs/ddeform_human.conf)? And what is the id  of the provided config sample (because deepdeform follow `seq<id>` format.)?

**Comment by RainbowRui** [comment]
1. We obtain masks by using off-the-shelf tools ([RobustVideoMatting](https://github.com/PeterL1n/RobustVideoMatting) for human data and [MiVOS](https://github.com/hkchengrex/MiVOS) for others).
2. You can try [the config sample](https://github.com/USTC3DV/NDR-code/blob/main/confs/ddeform_human.conf) on each DeepDeform sequence.

**Comment by phamtrongthang123** [comment]
Thank you. I want to ask one more question. Did you perform refinement (remove invalid points) in step 2 and 3 for Deep Deform data too?

**Comment by RainbowRui** [comment]
Yes, of course. It is an important step as [previous methods](https://github.com/Totoro97/NeuS/tree/main/preprocess_custom_data) shown.

**Comment by RainbowRui** [comment]
You are welcome.
